src/code/audio/audio_synthesis.c

```diff
@@ -1,137 +1,1705 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/audio_synthesis/func_801877D0.s")
+// DMEM Addresses for the RSP
+#define DMEM_TEMP 0x3B0
+#define DMEM_TEMP2 0x3C0
+#define DMEM_SURROUND_TEMP 0x4B0
+#define DMEM_UNCOMPRESSED_NOTE 0x570
+#define DMEM_HAAS_TEMP 0x5B0
+#define DMEM_COMB_TEMP 0x750             // = DMEM_TEMP + DMEM_2CH_SIZE + a bit more
+#define DMEM_COMPRESSED_ADPCM_DATA 0x930 // = DMEM_LEFT_CH
+#define DMEM_LEFT_CH 0x930
+#define DMEM_RIGHT_CH 0xAD0
+#define DMEM_WET_TEMP 0x3D0
+#define DMEM_WET_SCRATCH 0x710 // = DMEM_WET_TEMP + DMEM_2CH_SIZE
+#define DMEM_WET_LEFT_CH 0xC70
+#define DMEM_WET_RIGHT_CH 0xE10 // = DMEM_WET_LEFT_CH + DMEM_1CH_SIZE
```

Comment:
I'd quite like these written in terms of the other macros rather than just having it in a comment unless you think there's a good reason not to.

Reply:
Hey I said the same thing before
https://github.com/zeldaret/oot/pull/1289#discussion_r901153621
you have a tough crowd engineer

Reply:
Kinda building off my comment from that PR, I'd still say DMEM addresses are just carry-over and I still haven't properly documented what should and shouldn't line up. Does it make sense for `DMEM_WET_SCRATCH` to be exactly two channels after `WET_TEMP`, or is that just a happy coincidence? I'd like to understand dmem memory address management before moving away from hard-coded addresses